### PR TITLE
Modernize

### PR DESCRIPTION
**Pull request by tie-rack**
* Un-privatize vars
* Alias logging
* Remove resource-config dependency
* Remove unused reimann-clojure-client dependency
* Update to Clojure 1.7.0

### CODE REVIEW

**Comment by tie-rack** [comment]
This is work towards a version of this library more closely aligned with how we write libraries today.

File: project.clj [hunk 1]
```diff
@@ -3,11 +3,8 @@
   :url "http://github.com/turbovote/squishy"
```
Comment (cap10morgan) [inline project.clj]: Can you update that URL while you're in here and modernizing?

**Comment by cap10morgan** [comment]
Can you rename this to `democracyworks/squishy`?

File: project.clj [hunk 2]
```diff
@@ -3,11 +3,8 @@
...
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/clojure "1.6.0"]
+  :dependencies [[org.clojure/clojure "1.7.0"]
                  [com.amazonaws/aws-java-sdk "1.7.8.1"]
                  [org.clojure/tools.logging "0.2.6"]
                  [org.slf4j/slf4j-simple "1.7.7"]
```
Comment (cap10morgan) [inline project.clj]: Are the rest of the deps up to date?

File: src/democracyworks/squishy.clj [hunk 3]
```diff
@@ -1,44 +1,33 @@
...
-
-(defn- report-error [client body error]
-  (let [q (get-fail-queue client)]
-    (sqs/send client q (pr-str {:body body :error (.getMessage error)}))))
-
-(defn- safe-process [client f]
+(defn safe-process [client fail-queue-url f]
   (fn [message]
-    (info "Processing SQS message:" (str "<<" message ">>"))
+    (log/info "Processing SQS message:" (str "<<" message ">>"))
```
Comment (cap10morgan) [inline src/democracyworks/squishy.clj]: Should this be `log/debug` now?

**Comment by tie-rack** [comment]
@cap10morgan Renamed to demcoracyworks/squishy.

This is ready for another look. Near-future improvements will come faster with it. :)

**Comment by cap10morgan** [comment]
LGTM

**Pull request merged**